src/pages/Viewer/PageSlider.js

```diff
@@ -0,0 +1,60 @@
+import React, { useState } from 'react';
+import styled from 'styled-components';
+import theme from '../../styles/theme';
+
+const PageSlider = ({ currentPage, setCurrentPage, numPages }) => {
+  const [slidePage, setSlidePage] = useState(currentPage);
+
+  const handleOnchange = e => {
+    setSlidePage(e.target.value);
+  };
+
+  const handleInputSourceUP = () => {
```

Comment:
```suggestion
  const handleInputSourceUp = () => {
```
- 변수명은 카멜 케이스로 지어주세요
- InputSourceUp이라는 변수명이 직관적으로 이해되지 않는데 혹시 어떤 역할을 하는 함수인가요?

Reply:
하단 스크롤바(페이지 움직이는 기능) 컨트롤하는 함수입니다! 이름 handlePageMover로 변경했습니다